cmd/cloudexec/main.go

```diff
@@ -109,15 +109,16 @@ func main() {
 						return configErr
 					}
 					// Check if a local cloudexec.toml exists
-					if _, err := os.Stat(LaunchConfigFilePath); os.IsNotExist(err) {
+					launchConfigFilePath := c.String("config")
+					if _, err := os.Stat(launchConfigFilePath); os.IsNotExist(err) {
 						// Check if the path to a launch config is provided
 						if c.Args().Len() < 1 {
-							return fmt.Errorf("please provide a path to a cloudexec.toml file or create one in the current directory")
+							return fmt.Errorf("please create cloudexec.toml with 'cloudexec init' or use the '--config' flag to provide a path to your custom launch config file")
 						}
-						LaunchConfigFilePath = c.Args().Get(0)
+						launchConfigFilePath = c.String("config")
 					}
 					// Load the launch configuration
-					lc, err := LoadLaunchConfig(LaunchConfigFilePath)
+					lc, err := LoadLaunchConfig(launchConfigFilePath)
```

Comment:
The `config` flag was defined, mentioned in help text, but was completely ignored. It might surprise you by trying to load a launch config from the first positional arg, but that wouldn't work if other flags are provided first.

This change switches to using the `--config` flag as you'd expect based on help text.